src/main/resources/com/ncc/aif/restricted_hypothesis_aif.shacl

```diff
@@ -24,3 +24,32 @@ aida:SystemShape
             HAVING ((COUNT(?x)) != 1)
         """ ;
     ] .
+
+########################
+# 4.3 TA3 #7 Each entity (cluster) in the hypothesis graph must
+# have exactly one text description (called a "handle")
+#------------------------
+aida:HypothesisEntityClusterRequireHandle
+    a sh:NodeShape ;
+
+    sh:target [
+      a sh:SPARQLTarget ;
+      sh:select """
+            PREFIX aida:  <https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/InterchangeOntology#>
+            SELECT ?entityCluster
+            WHERE {
+                ?entityCluster a aida:SameAsCluster .
+                ?entityCluster aida:prototype ?prototype .
+                ?prototype a aida:Entity .
+            }
+        """ ;
+     ];
+
+    sh:property [
+        a sh:PropertyShape ;
+        sh:path aida:handle ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
```

Comment:
aida_ontology.shacl already enforces a maxCount. I recommend you either:
1. Disable the aida_ontology.shacl one (see aida:CompoundJustificationMinimum) so you can have a consistent, custom message.
1. Remove this maxCount and the message so that we use a consistent, default message

I'll be happy with either, so I'll leave it up to you

Reply:
I'll just disable the maxCount and make a comment that it is inherited from aida_ontology shacl.